tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "strictNullChecks": true,
     "noFallthroughCasesInSwitch": true,
     "noUnusedParameters": true,
-    "noUnusedLocals": true
+    "noUnusedLocals": true,
+    "lib": ["es2015"]
```

Comment:
@thewillhuang Did you try adding more things to `lib`? https://blogs.msdn.microsoft.com/typescript/2016/11/08/typescript-2-1-rc-better-inference-async-functions-and-more/ suggests you may want this:

```
        "lib": ["dom", "es2015.promise", "es5"]
```